src/coreclr/tools/metainfo/mdinfo.cpp

```diff
@@ -2039,7 +2040,7 @@ void MDInfo::DisplayCustomAttributeInfo(mdCustomAttribute inValue, const char *p
                 case ELEMENT_TYPE_U8:
                     CA.GetU8(&u8);
                     uI64 = u8;
-                    VWrite("%#lx", uI64);
+                    VWrite("%#" PRIu64, uI64);
```

Comment:
You can just use the Windows-specific `%#I64x` here (this code runs on the Windows emulator today, so the Windows-specific format string is fine) and then you won't need the `cinttypes` header that is causing the build breaks on Unix.